tests/test_raster.py

```diff
@@ -0,0 +1,11 @@
+"""Tests for the raster utilities module."""
+import doctest
+import unittest  # noqa: F401
+
+from aatoolbox.utils import raster
+
+
+def load_tests(loader, tests, ignore):
```

Comment:
I get the same coverage when I remove this function, do you find the same @caldwellst? 

I guess my one concern with the doctests is that they don't actually assert the method output. For example if I change `invert_coords` to not invert the coordinates at all, then this won't be picked up by the doc tests (I think).

Reply:
Son of a gun. Took me a long while to figure out, but clearly the doctests weren't running, because they do assert the method output (test by just right clicking in PyCharm and running). But, my solution to using doctest through unittest [doesn't work](https://docs.pytest.org/en/6.2.x/unittest.html).

> Up to this point pytest does not have support for the following features:
> - load_tests protocol
> - subtests

So, essentially, the `--with-doctests` argument passed to `pytest` is the recommended way, but gives us inflated coverage across non-tested modules. The `unittest` recommended way of defining `load_tests()` is actually not supported by `pytest`. So, I [found a simple way](https://stackoverflow.com/questions/5681330/using-doctests-from-within-unittests) to manually load those tests with `unittest` and it all works now!!